python/GafferUI/SpreadsheetUI/_EditWindow.py

```diff
@@ -57,15 +57,16 @@ def __init__( self, plugValueWidget, **kw ) :
 		self.keyPressSignal().connect( Gaffer.WeakMethod( self.__keyPress ), scoped = False )
 
 	@classmethod
-	def popupEditor( cls, plugs, plugBound ) :
+	def popupEditor( cls, plugs, plugBound, forceWindow = False ) :
 
 		if not isinstance( plugs, set ) :
 			plugs = { plugs }
 
 		plugValueWidget = GafferUI.PlugValueWidget.create( plugs )
 		cls.__currentWindow = _EditWindow( plugValueWidget )
 
-		if isinstance( plugValueWidget, _CellPlugValueWidget ) :
+		# Show a presets menu directly, as long as there is only one plug
+		if len( plugs ) == 1 and not forceWindow and isinstance( plugValueWidget, _CellPlugValueWidget ) :
```

Comment:
Somehow I'm managing to get a presets menu even with two plugs selected, but it's only editing the last one. This is the sequence :

- Make a column with a presets widget, and two rows.
- Click on cell in row1 to select it.
- Shift+click on cell in row2 to select it.
- Click on cell in row2 a second time, soon after the shift+click.

Why are we trying to limit to one plug anyway? PresetsPlugValueWidget supports multiple right?

Reply:
Resolving this one, since the second click on row2 now changes to single-selection, making the menu target obvious. And we can use right click to open an editor for the full selection if that's what we want.